python/py-cattrs/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-cattrs
+version             1.8.0
+revision            0
+platforms           darwin
```

Comment:
this is a default since MacPorts v2.7.0 so you can remove the `platforms` line (here and in the other Portfiles in this PR)

Reply:
is anything compiled in this package, if not you can probably add `supported_archs noarch`?